website/src/contribute/release-guide.md

```diff
@@ -618,6 +632,38 @@ Staging binaries
 
 Verify that files are [present](https://dist.apache.org/repos/dist/dev/beam).
 
+#### Stage SDK images on hub.docker.com
+Build Python images and push to DockerHub
+```aidl
+./gradlew :sdks:python:container:buildAll -Pdocker-tag=${RELEASE}_rc
+
+PYTHON_VER=("python2.7" "python3.5" "python3.6" "python3.7")
+for ver in "${PYTHON_VER[@]}"; do
+   docker push apachebeam/${ver}_sdk:${RELEASE}_rc &
+done
+``` 
+
+Build Java images and push to DockerHub
+```aidl
+./gradlew :sdks:java:container:dockerPush -Pdocker-tag=${RELEASE}_rc
+```
+
+Build Go images and push to DockerHub
+```aidl
+./gradlew :sdks:go:container:dockerPush -Pdocker-tag=${RELEASE}_rc
+```
+
+Clean up images from local
+```aidl
+for ver in "${PYTHON_VER[@]}"; do
+   docker rmi -f apachebeam/${ver}_sdk:${RELEASE}_rc
+done
+docker rmi -f apachebeam/java_sdk:${RELEASE}_rc
+docker rmi -f apachebeam/go_sdk:${RELEASE}_rc
+```
+
+Verify all images are publish to DockerHub with tags :${RELEASE}_rc
```

Comment:
Can you also provide a link/simple instruction to find those images on hub.docker.com?